### DISCUSSION TITLE
Questions about APP_VERSION=*options*

### DISCUSSION THREAD

**Discussion by Spacellary** [nikhilbadyal/docker-py-revanced#143] [General]
Heya! Thanks a lot for this project, it is very convenient to build the apps quickly and from all platforms.

I have a question about the APP_VERSION= option in the .env file.

I am building Extended, and by default without any options it will download and patch the latest stable YouTube APK version from APKMirror, if I include the `latest` option, but including Betas, now, is there a parameter like "recommend" or "supported" for the APP_VERSION= option?

Since no matter what it will most likely build an *unsupported* version since the chances of a newer stable .apk being available are quite high, even though Extended usually has a very recent supported version it still happens, and is well, a lot worst for people building official ReVanced where it will pretty much always build an "unsupported" version by default.

I know you can manually specify the version, but is there a parameter that automatically will build just the latest **supported** version instead of just the latest? The support versions are listed in the patches.json file.

It takes away the ease of use from the "one click" nature of the tool, and if having to manually set options for each new version build, it doesn't feel much different than just doing the manual CLI builds, I think at least.

**Comment by nikhilbadyal** [comment]
Just **don't** add anything(related to version) and it will build the recommended version.

In the **Build Yourself** section.  
If you check 3rd step in building through GitHub. It says 

`Add .envs if you want to cook something else.`


Also I'll add this one explicitly if it's confusing.

**Reply by Spacellary** [reply]
**Thanks for the clarification!**
I should have assumed that was the default behavior but just wanted to be sure.
